iOS Example Frame/iOS Example Frame/Factory/Factory.swift

```diff
@@ -56,6 +56,9 @@ struct Factory {
   }
 
   static func getMatrixPaymentViewController(completionHandler: @escaping (Result<TokenDetails, TokenisationError.TokenRequest>) -> Void) -> UIViewController {
+    #if UITEST
+    return getCompleteUITestVC(completionHandler: completionHandler)
+    #endif
```

Comment:
this is code is for merchants. it should not include UITest logic

Reply:
This is a demo app which comes with UITests. 
This is not code for merchants as it does not form the SDK consumer would use, its built on top of it.

Reply:
It's an Example app and they will reuse code. We may discuss this with team in next standup